components/bms_boss/src/Sys.c

```diff
@@ -114,7 +114,7 @@ void SYS_SFT_openContactors(void)
     SYS.contacts = SYS_CONTACTORS_OPEN;
 }
 
-void SYS_SFT_cycleContacts(void)
+void closeSYS_SFT_cycleContacts(void)
```

Comment:
Why did you change the name of this? This function does not close the contacts. It cycles the contacts state